.github/workflows/check_smoke_tests.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           - 1433:1433
         env:
           ACCEPT_EULA: Y
-          SA_PASSWORD: 'YourStrong!Passw0rd'
+          SA_PASSWORD: "YourStrong!Passw0rd"
 
     steps:
       - name: Check out repository
```

Comment:
Do we also need to override INITIAL_ALGORITHMS on line 72?  I guess that depends on if we want to verify everything is working with placeholder values or if we should be verifying that it works with real values.  I can see benefits in both approaches.